docs/user_guide/usage_patterns/data_transfer/transfer_relative_deadline/index.rst

```diff
@@ -0,0 +1,103 @@
+.. currentmodule:: globus_sdk
+
+.. _userguide_transfer_relative_deadline:
+
+Setting a Relative Deadline for a Transfer
+==========================================
+
+When transferring or deleting data via Globus Transfer, users are able to set a
+``deadline`` for their tasks.
+This allows you to declare a time by which the task must be completed -- if the
+deadline is reached and the task is still in progress, it will be cancelled.
+
+The ``deadline`` field in a Transfer task takes a date and time in ISO 8601
+format. Because of the use of a standard format, it is easy to use the Python
+``datetime`` module to compute a relative deadline at some point in the future.
+You can use this to easily submit tasks with deadlines limited to the next minute,
+hour, or day.
+You can use this, for example, to enforce that a Transfer Task which takes too
+long results in errors (even if it is making slow progress).
+
+For readers who prefer to start with complete working examples, jump ahead to the
+:ref:`example script <userguide_transfer_relative_deadline_example>`.
+
+Computing and Formatting a Deadline
+-----------------------------------
+
+We need to compute a relative deadline -- some point into the future -- and
+format it as a string.
+We'll express that idea in a function which takes a :class:`datetime.timedelta`
+as an ``offset``, an amount of time into the future.
+This gives us a generic phrasing of getting a future date:
+
+.. code-block:: python
+
+    import datetime
+
+
+    def make_relative_deadline(offset: datetime.timedelta) -> str:
+        now = datetime.datetime.now(tz=datetime.UTC)
```

Comment:
`datetime.UTC` is an alias that is only available on Python 3.11 and higher. This needs to be `datetime.timezone.utc` for broad compatibility.

```suggestion
        now = datetime.datetime.now(tz=datetime.timezone.utc)
```

Reply:
Ach! Thanks for this catch. I was overly focused on my local dev environment.